src/pages/embedded/index.ts

```
@@ -840,132 +928,68 @@ window.addEventListener("DOMContentLoaded", function () {
-						for (const [featureName, { new: newPlacement, old: oldPlacement }] of Object.entries(singleButtonChanges)) {
-							if (oldPlacement === newPlacement) continue;
-							const featureFuncs = getFeatureFunctions(featureName, oldPlacement);
-							await featureFuncs.remove();
-							await featureFuncs.add();
-						}
 						break;
 					}
-					case "shortsAutoScrollChange": {
+					default: {
+						return;
+					}
```

Biome (1.9.4)

[error] The default clause should be the last switch clause.

The following case clause is here:

Regardless its position, the default clause is always executed when there is no match. To avoid confusion, the default clause should be the last switch clause.

(lint/suspicious/useDefaultSwitchClauseLast)